### DISCUSSION TITLE
Welcome to vsRobloxButNotReally Discussions!

### DISCUSSION THREAD

**Discussion by mayo78** [mayo78/vsRobloxButNotReally#1] [Announcements]
post the stuff you make here :)

**Comment by NokioPyro** [comment]
Alright, makin a credit.txt file rk with icons lmao

**Comment by NokioPyro** [comment]


**Reply by ThatOneSillyGuyDon** [reply]
Noice.

**Comment by ThatOneSillyGuyDon** [comment]
I literally had to update my status cause people who come from GitHub keep spamming my Discord DMs to help with a mod that probably isn’t good anyways
![1A2B2374-FF3C-40BA-AE75-B240B074370B](https://user-images.githubusercontent.com/87618246/160499575-039a2fc8-c9fa-459d-a2be-b35d4123c818.jpeg)

**Reply by Sticksaga** [reply]
my icon: 
![icon-hayden](https://user-images.githubusercontent.com/96336265/166817731-2285094b-1f13-44d8-be26-d5a671a52e86.png)
